docs/cloud/aws/getting-started/aws-console-access.md

```diff
@@ -1,19 +1,19 @@
 # Accessing the AWS Management Console
 
 ## Prerequisites
-1. You are a member of CUBoulder. If you have access to CUBoulder AWS resources and are from another institution, please see <a href="./aws-console-access-non-cu-boulder.md" target="_blank">this guide</a> on signing in using your institution's credentials.
+1. You are a member of CUBoulder. If you have access to CUBoulder AWS resources and are from another institution, please see [this guide](non-cu-boulder/aws-console-access-non-cu-boulder.html) on signing in using your institution's credentials.
```

Comment:
For internal references, you should not use `.html`, instead, `.md` should be used.

Reply:
ccb0f5c